src/hooks/useActiveDescendantListFocus/__tests__/useActiveDescendantListFocus.jest.js

```diff
@@ -93,22 +170,57 @@ function runListUnitTest(isHorizontal) {
       userEvent.keyboard(oppositeMoveForwardKey);
     });
 
-    expect(result.current.visualFocusItemIndex === 0).toBeFalsy();
+    expect(result.current.visualFocusItemIndex).not.toEqual(naturalIndex + 1);
   });
 }
 
+const combineFeatures = features => {
+  const combinations = [];
+  // all closed
+  const allClosedComb = {};
+  for (let feature of features) {
+    allClosedComb[feature] = false;
+  }
+  combinations.push(allClosedComb);
+
+  // all open
+  const allOpenComb = {};
+  for (let feature of features) {
+    allOpenComb[feature] = true;
+  }
+  combinations.push(allOpenComb);
+
+  // one open all other closed
+  for (const feature of features) {
+    const oneOpen = {};
```

Comment:
this is super cool but now u will have tests that will run only for specific feature

Reply:
You have either all open or all closed.

and also one open each time.

Didnt want to make it expanontially o(2^n) I can do it o(n^2) though so each feature will be with another feature, but I think this way is enough and each feature will be together anyway since there is an option where all are together just not two alone

Reply:
See here
![image](https://user-images.githubusercontent.com/2057934/178156202-b6e83e81-6a31-4525-aeb0-891b5a3d1404.png)